pkg/ccl/changefeedccl/cdcevent/event.go

```diff
@@ -287,6 +293,12 @@ func NewEventDescriptor(
 		}
 	}
 
+	allCols := make([]int, 0)
```

Comment:
this is the same as `var allCols []int`
Or... you can `allCols := make([]int, len(sd.cols)` and then set `allCols[i] = i`